RucioClient.py

```diff
@@ -91,8 +91,10 @@ def getFileCountPerBlock(self, dataset):
             blocks = []
             for block in self.getBlockNamesDataset(dataset):
                 blocks.append((block, self.getFileCountBlock(block)))
+            if blocks is None:
+                raise Exception("number of file count per block is None")
         except Exception as e:
             print(str(e))
-            return []
+            return 0
```

Comment:
the function getFileCountPerBlock needs to return a list, empty if there's an exception

Reply:
the function returns the number of files not a list as the description explains below the function. https://github.com/CMSCompOps/WmAgentScripts/blob/master/RucioClient.py#L74
https://github.com/CMSCompOps/WmAgentScripts/blob/master/RucioClient.py#L85

Reply:
I see your point. my bad